src/main/java/net/wasdev/wlp/common/plugins/util/DevUtil.java

```diff
@@ -211,6 +211,7 @@
     private String applicationId;
     private int appUpdateTimeout;
     private Thread serverThread;
+    private boolean devStop;
```

Comment:
Since multiple threads (the main DevUtil thread and the shutdown hook thread) can modify this value, we should use something like AtomicBoolean.